app/Controllers/categoryController.php

```diff
@@ -47,7 +47,7 @@ public function createAction() {
 
 			$cat = new FreshRSS_Category($cat_name);
 
-			if ($catDAO->searchByName($cat->name()) != null) {
+			if ($catDAO->searchByName($cat->name()) !== null) {
```

Comment:
See https://github.com/FreshRSS/FreshRSS/pull/4439/files#r919511685
Also, it should probably be in the same PR